sysutils/bupstash/Portfile

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        andrewchambers bupstash 0.6.4 v
+revision            0
+
+homepage            https://bupstash.io
+
+description         Easy and efficient encrypted backups
+
+long_description    Bupstash is a tool for encrypted backups - if you need \
+                    secure backups, Bupstash is the tool for you.  Bupstash \
+                    was designed to have: Efficient deduplication, Offline \
+                    decryption keys, Key/value tagging with search, Great \
+                    performance on slow networks, Secure remote access \
+                    controls, Efficient incremental backups, Fantastic \
+                    performance with low RAM usage, Safety against malicious \
+                    attacks.
+
+categories          sysutils
+platforms           darwin
+license             MIT
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+checksums           ${distname}${extract.suffix} \
+                    rmd160  01417e673865bebb744c71af26ce07e2c9b297a3 \
+                    sha256  1cfeae63cab2a2e6bb45b0f260b12a4f89acb64b51a43b36e4573c7a46289cab \
+                    size    119320
+
+depends_lib-append  port:libsodium \
+                    port:pkgconfig
```

Comment:
Just looking through the [Cargo.toml](https://github.com/andrewchambers/bupstash/blob/fd0eb84557bd72ecbb2b7a10add911a8f4ec56a6/Cargo.toml) file, pkgconfig is listed as a build dep only.

I think it should be safe to just put it in a `depends_build` block, but please correct me if I'm wrong on this.

Reply:
You're 100% right, great catch